packages/nimble-components/CONTRIBUTING.md

```diff
@@ -332,6 +332,14 @@ const nimbleButton = Button.compose({
 });
 ```
 
+### Leverage mixins for shared APIs across components
+
+Typescript and the FAST library each offer patterns and/or mechanisms to alter the APIs for a component via a mixin.
+
+FAST provides an `applyMixins` function (which is just an implementation of the pattern described [here](https://www.typescriptlang.org/docs/handbook/mixins.html#alternative-pattern) in the Typscript docs) to alter the API of a given component with a set of provided mixin classes. This is used in several places within Nimble (see [here](https://github.com/ni/nimble/blob/6839ee05cf4d72efa6a20cd23e1d830047103745/packages/nimble-components/src/toggle-button/index.ts#L44) and [here](https://github.com/ni/nimble/blob/6839ee05cf4d72efa6a20cd23e1d830047103745/packages/nimble-components/src/anchor-tabs/index.ts#L274)).
```

Comment:
Provide [descriptive link text](https://webaim.org/techniques/hypertext/link_text#uninformative) to improve readability of documentation generally for all readers and with the additional benefit of improved accessibility.

The way I train myself is that the link text itself should be descriptive enough standalone but shouldn't be a blue paragraph (just a couple words long), i.e:

> See how the [ToggleButton `StartEnd` mixin](#) is applied.

On its own it Is much easier to read documentation and gives me standalone context about what to expect when following the link. Also, you may remember that screen reader users had that shortcut to quickly navigate all links in a page, so multiple links with the link text `here` aren't helpful.